ui/app/hooks/useCancelTransaction.js

```diff
@@ -0,0 +1,50 @@
+import { useDispatch, useSelector } from 'react-redux'
+import { useCallback } from 'react'
+import { showModal } from '../store/actions'
+import { isBalanceSufficient } from '../pages/send/send.utils'
+import { getHexGasTotal, increaseLastGasPrice } from '../helpers/utils/confirm-tx.util'
+import { getConversionRate, getSelectedAccount } from '../selectors'
+
+/**
+ * useCancelTransaction
```

Comment:
Nit: can we add a single line description of the function here instead? Maybe extracted from the larger description below.

Reply:
```
/**
 * Determine whether a transaction can be cancelled and provide a method to
 * kick off the process of cancellation.
```